src/api/app/views/layouts/webui/responsive_ux/_nobody_navigation.html.haml

```diff
@@ -1,2 +1,22 @@
-= sign_up_link(css_class: 'nav-link text-light text-center p-1 toggler')
-= log_in_link(css_class: 'nav-link text-light text-center p-1 toggler')
+.toggler
+  = sign_up_link(css_class: 'nav-link text-light text-center p-1')
+.toggler
+  = log_in_link(css_class: 'nav-link text-light text-center p-1')
+.toggler
+  = link_to('javascript:void(0)', class: 'nav-link text-light', alt: 'Generic Links', data: { toggle: 'offcanvas' }) do
+    %i.fas.fa-user-circle.fa-2x
```

Comment:
Sorry, I didn't realized before this icon was added when nobody is logged in.
Why don't we use a "hamburguer" menu icon instead? It's content is not related to a user at all, as we can see in this image:
![Screenshot_2020-03-10 Open Build Service](https://user-images.githubusercontent.com/2581944/76310780-c6209d80-62cf-11ea-8371-6208504a03cd.png)

Reply:
In fact, I would say that when logged in, the gravatar should be replaced by the "hamburger" menu also. Since it contains links like "Status Monitor" and not only user-related things. But it can be done in another PR.

Reply:
I agree! I'll do it now :)

Reply:
So it's okay to address those 2 changes in a separate PR?

Reply:
sure!